src/Admin/BlockAdmin.php

```diff
@@ -231,20 +229,6 @@ private function configureBlockFields(FormMapper $form, BlockInterface $block):
             } else {
                 $service->configureCreateForm($blockMapper, $block);
             }
-        } else {
-            @trigger_error(
```

Comment:
According to the deprecation you have to change the check ligne 207 to "instanceof Editable"

Reply:
Don't know what you mean, the if already have an instanceof check, right?

Reply:
(Im on phone)

The code is

If not instanceof Blockeditableservice, throw exception Foo

If instanceof EditableBlock, things,
Else deprecation

This must become

If not instanceof EditableBlock,
Throw exception Foo

And then things

Dunno if Im clearer or ill need to access a computer next week

Reply:
Fixed.